scene/resources/font.cpp

```diff
@@ -138,7 +138,6 @@ bool Font::_is_cyclic(const Ref<Font> &p_f, int p_depth) const {
 		}
 		return _is_cyclic(f, p_depth + 1);
 	}
-	return false;
```

Comment:
That's definitely not true, this return is the most common path, not unreachable. MSVC probably is confused by recursive call.

Reply:
Ah indeed I missed that this was a recursive function (and indeed even then it should be reachable if `p_f->fallbacks.size()` is 0).

I'll remove that change from this PR for now.